src/mock-doc/document.ts

```diff
@@ -172,7 +172,7 @@ export class MockDocument extends MockHTMLElement {
   }
 
   createElementNS(namespaceURI: string, tagName: string) {
-    const elmNs = new MockElement(this, tagName);
+    const elmNs = createElement(this, tagName);
```

Comment:
if I am not wrong createElementNS() always creates MockElement (or svg) but never HTMLElement

Reply:
<img width="466" alt="Screenshot 2019-06-28 at 22 30 04" src="https://user-images.githubusercontent.com/127379/60369557-4af2b200-99f4-11e9-9997-e590e5bcc47b.png">

Reply:
The way it was before, it always created a MockElement...

Do we need to abstract out some of the logic from `createElement()` so it can create a MockSVGElement when required?